homeassistant/components/aprs/device_tracker.py

```diff
@@ -8,7 +8,9 @@
 import geopy.distance
 import voluptuous as vol
 
-from homeassistant.components.device_tracker import PLATFORM_SCHEMA
+from homeassistant.components.device_tracker import (
+    PLATFORM_SCHEMA as PLATFORM_SCHEMA_BASE,
```

Comment:
We shouldn't use the name `PLATFORM_SCHEMA_BASE` as that has a specific function in our config validation.

https://github.com/home-assistant/core/blob/7350942e9e2d68803c5e51c7d6796c4234363b7d/homeassistant/config.py#L817

Reply:
I'm thinking about better name and I come up with these options:
- `BASE_PLATFORM_SCHEMA`
- `PARENT_PLATFORM_SCHEMA`

Which one do you prefer?